hack/quickstart/init-master.sh

```diff
@@ -70,31 +70,36 @@ function init_master_node() {
     exit 1
 }
 
-# This script can execute on a remote host by copying itself + kubelet service unit to remote host.
+# This script can execute on a remote host by copying itself + kubelet service unit + auto_approver to remote host.
 # After assets are available on the remote host, the script will execute itself in "local" mode.
 if [ "${REMOTE_HOST}" != "local" ]; then
     # Set up the kubelet.service on remote host
     scp -i ${IDENT} -P ${REMOTE_PORT} ${SSH_OPTS} kubelet.master core@${REMOTE_HOST}:/home/core/kubelet.master
     ssh -i ${IDENT} -p ${REMOTE_PORT} ${SSH_OPTS} core@${REMOTE_HOST} "sudo mv /home/core/kubelet.master /etc/systemd/system/kubelet.service"
 
-    # Copy self to remote host so script can be executed in "local" mode
+    # Copy the auto approver to remote host and execute it for 1h
+    scp -i ${IDENT} -P ${REMOTE_PORT} ${SSH_OPTS} ../auto_approver.sh core@${REMOTE_HOST}:/home/core/auto_approver.sh
+    ssh -i ${IDENT} -p ${REMOTE_PORT} ${SSH_OPTS} core@${REMOTE_HOST} "/home/core/auto_approver.sh 3600" &
```

Comment:
Not sure if I get what you just suggested?

I believe we should not have an auto-approver by default in Bootkube elsewhere than in the `hack/` examples. Production users should have control over which nodes are allowed to join, and be able to define their own approval policy, regardless if it is the first or the 10th node.

We could however simply have that script as a systemd unit on masters? Or maybe bootkube should have an integrated command to manage approvals (list, approve, reject, auto-approve).. since kubectl doesn't have one.